projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.ts

```diff
@@ -103,11 +107,16 @@ export class ManageRulesSmartComponent implements OnInit, OnDestroy {
         this.folderRuleSetsService.loadRuleSets(this.nodeId);
       }
     });
+
+    this._actionDefinitionsSub = this.actionDefinitions$.subscribe((actionDefinitions) =>
```

Comment:
Same here can we specify type of `actionDefinitions`

Reply:
Done, ty.